lib/tasks/dev.rake

```diff
@@ -5,8 +5,22 @@ if Rails.env.development? || Rails.env.test?
     desc "Sample data for local development environment"
     task prime: "db:setup" do
       include FactoryBot::Syntax::Methods
+      User.delete_all
 
-      # create(:user, email: "user@example.com", password: "password")
+      User.create(
```

Comment:
A side note: I usually prefer to use: `where(email: "some_email").first_or_create` to make sure we are not creating objects again and again. Doesn't matter here because we are deleting all the users. But something to remember when you are working with a database you cannot wipe out every time.

Reply:
Thanks Elle